docs/pages/get-started/faq.md

```diff
@@ -466,27 +544,43 @@ Different approaches to long-term message storage are currently being researched
 
 ## Messages
 
-### Which message formats and metadata does XMTP support?
+### Which message formats does XMTP support?
 
-XMTP transports a message payload as a set of bytes that can represent any format that a developer wants to support, such as plain text, JSON, or even non-text binary or media content.
+XMTP transports a message payload as a set of bytes that can represent any format that a developer wants to support, such as plain text, JSON, or non-text binary or media content.
 
 With XMTP, these message formats are called content types. Currently, there are two basic content types available. These content types aim to establish broad compatibility among apps built with XMTP.
 
 The XMTP community can propose and adopt standards for other content types, either informally or through a governance process.
 
-Message payloads also include references to timestamps. However, timestamps are not currently independently verified and can be set to any value by the sending app.
-
 To learn more about content types, see [Content types](/content-types/content-types).
 
 To learn more about the XMTP improvement proposals governance process, see [What is an XIP?](https://github.com/xmtp/XIPs/blob/main/XIPs/xip-0-purpose-process.md)
 
-Have other questions or ideas about message formats and metadata? Post to the [XMTP Community Forums](https://community.xmtp.org/).
+### Which message metadata does XMTP support?
```

Comment:
Here's where Cursor hallucinated about how XMTP handles metadata in v3 vs v2. I thought it was interesting - so sharing in this PR.

Would love to understand if there are differences in metadata handling in v2 vs v3.